### ISSUE TITLE
[Slider] Wrong calculation of min-width of slider fill element on changing both handle values programmaticaly

### ISSUE THREAD

**Issue by Duskyman** [zurb/foundation-sites#9124]
**How can we reproduce this bug?**

Recreate a twoHandles slider with different initialStart and initialEnd options.

**What did you expect to happen?**

<span class="slider-fill"> element should fill the whole space between handles as it is supposed to do(not only partially or exceeding beyond right handle)

**What happened instead?**

the element calculates its min-width wrong(from the new position of left handle to the old position of right handle).

**Test case:**

Look at this fiddle: https://jsfiddle.net/bpr2bjch/

Every second, the values of slider change randomly. Since there is no API(except for _setHandlePos(), which is private) for getting/setting the values of the slider, I am recreating the slider with different initialStart and initialEnd options.

The problem is caused by the wrong calculation of css min-width property on line 169(https://github.com/zurb/foundation-sites/blob/v6.2.2/js/foundation.slider.js#L169).

Here, the min-width of the slider-fill element is calculated from css property of the handle, but the right handle didn't moved yet(because animation), so the calculation is wrong. I managed to fix this issue with this two lines:
`var location2 = ((elemDim - handleDim) * percent(parseFloat(this.$handle2.attr('aria-valuenow')) - this.options.start, this.options.end - this.options.start) / elemDim * 100).toFixed(2);
dim = parseFloat(location2) - movement + handlePct;`

Basically, I am calculating the final position of the handle, where it will sit after the animation is done and swapping it with the value gained from css property on the next line.

**Comment by kball** [comment]
This was fixed in #9280,
